wiki/Skinning/de.md

```diff
@@ -0,0 +1,30 @@
+# Skinning
+
+Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen, um ihre eigenen zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
+
+## Skinsets
+
+*Haupseite: [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+Wenn du einen speziellen Skin in deiner Beatmap verwendest, wird dies als Beatmap-spezifisches Skinning bezeichnet. Hierbei werden die einzelnen Elemente des Skins direkt in den Beatmap-Ordner gepackt und dadurch im Spiel verwendet sofern der Spieler dies nicht ausdrücklich anders eingestellt hat. Wenn du einen solchen Skin verwenden möchtest, ist das [Ranking Criteria](/wiki/Ranking_Criteria) sowie die [Skinsetliste](/wiki/Ranking_Criteria/Skin_Set_List) zu beachten. 
```

Comment:
uhhh i think i need a comma between
`verwendet, sofern `

but am not sure

Reply:
I'd put a comma between "verwendet" and "sofern". So basically: [...] und dadurch im Spiel verwendet, sofern der Spieler dies nicht [...]